low-level/sensors/impl/centos_7/hpi_monitor.py

```diff
@@ -52,6 +52,12 @@ def name():
         """@return: name of the monitoring module."""
         return HPIMonitor.SENSOR_NAME
 
+    @staticmethod
+    def impact():
```

Comment:
Suggestion:- we can create a mapping also with service and it's impact str. instead of implementing impact() func in every service file.

Reply:
This is per sensor property, so it's okay to have per class I guess. I was thinking of `@property` decorator if applies instead of `@staticmethod`

Reply:
Since impact() is sensor specific, I kept like this.
The problem with mapping is, we need to align module name future changes separately even though impact() doesn't have any change. Again we will lose this class specific attributes.
```
sspl_constants.py
{
    "MemFaultSensor": "Server memory can not be monitory"
}
```

Reply:
I kept the recommendation is, "Restart SSPL service". Nothing specific to sensor. So it is generic and directly used in thread_monitor_msg.py.
I will look into this point after addressing all other comments.